src/main/java/seedu/letsgethired/model/application/Note.java

```diff
@@ -11,10 +11,11 @@ public class Note {
 
     public static final String MESSAGE_CONSTRAINTS =
             "Note insertion must be followed by the prefix 'i/' "
-                    + "and should only contain characters and spaces, and it should not be blank\n"
+                    + "and should not contain any slashes, and it should not be blank\n"
```

Comment:
```suggestion
                    + "and should not contain any slashes (/), and it should not be blank\n"
```
Perhaps we could specify what slashes are? Just in case the user is unfamiliar with it!

Reply:
Back slashes and forward slashes in particular